arch/risc-v/src/mpfs/mpfs_plic.h

```diff
@@ -58,7 +58,7 @@ void mpfs_plic_init_hart(uintptr_t hartid);
  *
  ****************************************************************************/
 
-uintptr_t mpfs_plic_get_iebase(void);
```

Comment:
Please keep the incoming parameter as uintptr_t 

I see that the conversion macros have wrong types (they use int for hartid), but let's not spread that rot here as well.

Hartid should be uintptr_t as its width depends on the architecture (32-bits on RV32 and 64-bits on RV64)